### PR TITLE
Feat review count update realtime

### LINKED ISSUE #73: "Be the first to review" should disappear after first review

### PR DESCRIPTION
**Pull request by mxng**
## Why

User review on the restaurant page is not updating realtime when user submits a new review

## What

This pull request introduces the following:

 * Now it does

### Others

There's a problem though for subsequent new reviews that the site is not able to find the message to delete.
Doesn't affect the performance.

Screenshot 2021-09-27 at 3 59 27 PM

### CODE REVIEW

**Comment by mxng** [comment]
[Let review count update REAL TIME when new reviews are added](https://trello.com/c/r2LW707w/133-let-review-count-update-real-time-when-new-reviews-are-added)

**Comment by mxng** [comment]
[Let review count update REAL TIME when new reviews are added](https://trello.com/c/r2LW707w/133-let-review-count-update-real-time-when-new-reviews-are-added)

**Comment by mxng** [comment]
[Let review count update REAL TIME when new reviews are added](https://trello.com/c/r2LW707w/133-let-review-count-update-real-time-when-new-reviews-are-added)

**Review by tjw0612 — APPROVED** [review]
looks good

File: app/javascript/controllers/insert_in_list_controller.js [hunk 1]
```diff
@@ -17,6 +17,16 @@ export default class extends Controller {
...
         if (data.inserted_item) {
           this.itemsTarget.insertAdjacentHTML(this.positionValue, data.inserted_item);
+
+          // Update review count
+          const reviewCount = document.querySelector("#review-count");
+          const count = parseInt(reviewCount.innerText, 10) + 1;
+          reviewCount.innerText = count;
+
+          // Remove 0 reviews notice msg
+          if (this.zeroReviewsNoticeTarget) {
```
Comment (TeeWhyKay) [inline app/javascript/controllers/insert_in_list_controller.js]: do check with how to solve the console issue 👍

**Review by TeeWhyKay — APPROVED** [review]
Look fine!

**Review by amelialimyh — APPROVED** [review]
Looks fine

**Review by tjw0612 — APPROVED** [review]
nice work for the query selector

**Review by TeeWhyKay — APPROVED** [review]
Thanks for updating!

**Pull request merged**